skipper.go

```diff
@@ -868,6 +869,7 @@ func Run(o Options) error {
 		PostProcessors: []routing.PostProcessor{
 			loadbalancer.HealthcheckPostProcessor{LB: lbInstance},
 			loadbalancer.NewAlgorithmProvider(),
+			lifo.NewRegistry(lifo.Config{}, nil),
```

Comment:
Maybe I missed it, but this will just use zero values for the Config, isn't it?
If that's the case, I'd rather have a DefaultConfig with sane default settings and apply it here.

Reply:
in its current form, that's correct. It needs to be 'wired' in the main options and the cmd line args.

The default config is still a good idea, we just need to collect first some measured data. Even more challenging is to gain some experience that supports decisions like: "if my max concurrency is this, then what is the right max stack size, and then what timeouts to use".

Reply:
I dropped the whole global config, because of dependency cycles it was not easily possible to achieve it. I think it also not worth the effort, because you can achieve the global style with default filters, for example:  ` -default-filters-prepend='lifo(100,20,"10s")'`